FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Shashank Mishra on 25/04/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	static private var OK_200 = 200
```

Comment:
To be consistent with the rest of the code the `private` keyword should precede `static` (access level comes first):
```suggestion
	private static var OK_200 = 200
```

Reply:
Updated